typespec/Applications.Core/applications.tsp

```diff
@@ -39,23 +39,23 @@ namespace Applications.Core;
 
 @doc("Radius Application resource")
 model ApplicationResource is TrackedResource<ApplicationProperties> {
-  @doc("The application name")
+  @doc("The name of your resource.")
   @path
   @key("applicationName")
   @segment("applications")
   name: ResourceNameString;
 }
 
-@doc("Application properties")
+@doc("Properties of the resource.")
 model ApplicationProperties {
   @doc("The status of the asynchronous operation.")
   @visibility("read")
   provisioningState?: ProvisioningState;
 
-  @doc("Fully qualified resource ID for the environment that the portable resource is linked to")
+  @doc("The ID of the environment resource this container belongs to.")
```

Comment:
Shouldn't this be application instead of container?

Reply:
Yeah possibly but currently that's the definition we have here: https://edge.docs.radapp.io/reference/resource-schema/core-schema/application-schema/#properties I think this PR might highlight a lot of descriptions we might want to change.

Reply:
Yep this should be application. Typo in the current docs.

Reply:
```suggestion
  @doc("The ID of the environment resource this application belongs to. Specified in the fully-qualified UCP ID format.")
```